targets/gamepad/include/FeatureFlags.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
```

Comment:
So is the intent to have all uwrt feature flag in `FeatureFlags.h` and have all mbed feature flags in `mbed_config_target.h` and the other shared mbed config header?

Reply:
I think that makes sense. The uwrt configs should be separate from the mbed configs since the mbed header gets included in a portion of the mbed build that I think uses a C compiler? Cuz I couldn't compile the namespaced feature flags when I put them in `mbed_config_target.h`

Reply:
Super nit: 
Since we have `mbed_config_target`, how about `uwrt_config_target`

Reply:
Also yeah I vaguely remembered needing to do that because of how mbed sets up their stuff